src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -87,6 +88,14 @@ public static void copyInventoryItems(IItemHandler src, IItemHandlerModifiable d
         }
     }
 
+    public static InventoryCrafting deepCopyInventoryCrafting(final InventoryCrafting source) {
+        final InventoryCrafting result = new InventoryCrafting(new DummyContainer(), source.getWidth(), source.getHeight());
+        for (int i = 0; i < result.getSizeInventory(); ++i) {
```

Comment:
Please change implementation to use `i++` instead of `++i` as it is more common through project.